docs/core/tools/dotnet-store.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
again, review the metadata section from current topics. also ms.date should be more current than october of last year.